http-gateway/web/src/containers/things/things-i18n.js

```diff
@@ -128,14 +138,30 @@ export const messages = defineMessages({
   },
   resourceCreateError: {
     id: 'things.resourceCreateError',
-    defaultMessage: 'Resource create failed',
+    defaultMessage: 'Failed to create a resource',
     description: 'Title of the toast message on resource create error.',
   },
   resourceRetrieveError: {
     id: 'things.resourceRetrieveError',
-    defaultMessage: 'Resource retrieve failed',
+    defaultMessage: 'Failed to retrieve a resource',
     description: 'Title of the toast message on resource retrieve error.',
   },
+  resourceDeleteSuccess: {
+    id: 'things.resourceDeleteSuccess',
+    defaultMessage: 'Resource delete scheduled',
+    description:
+      'Title of the toast message on delete resource schedule success.',
+  },
+  resourceWasDeleted: {
+    id: 'things.resourceWasDeleted',
+    defaultMessage:
+      'Resource delete was successfully scheduled. Keep the notifications turned on to see if the resource was deleted.',

```

Comment:
`The resource delete was successfully scheduled. you will be notified when the resource was deleted.`